src/System.Collections/tests/Generic/Stack/Stack.Tests.cs

```diff
@@ -31,16 +31,19 @@ protected override ICollection NonGenericICollectionFactory()
         /// <summary>
         /// Returns a set of ModifyEnumerable delegates that modify the enumerable passed to them.
         /// </summary>
-        protected override IEnumerable<ModifyEnumerable> ModifyEnumerables
+        protected override IEnumerable<ModifyEnumerable> GetModifyEnumerables(ModifyOperation operations)
         {
-            get
+            if ((operations & ModifyOperation.Add) == ModifyOperation.Add)
             {
                 yield return (IEnumerable enumerable) =>
                 {
                     var casted = (Stack<string>)enumerable;
                     casted.Push(CreateT(2344));
                     return true;
                 };
+            }
+            if ((operations & ModifyOperation.Remove) == ModifyOperation.Remove)
```

Comment:
Style, feel free to ignore: writing like `((operations | ModifyOperation.Remove) != 0)` is shorter/cleaner maybe.

Reply:
I think (assume?) this is the most common way in our codebase

Reply:
You’d still want bitewise and :).  Any value bitwise-or nonzero is nonzero :)

Reply:
@Wraith2 - actually, it would somewhat _prevent_ that error, since you're only mentioning the operation once.  You then just have the normal "did this branch show up twice" error.

I sortof want to say we want some sort of extension method for this: `someEnum.IsSet(someEnumOption)` that does the proper check under the covers, which would cover a lot of these error types.  Should be inlineable, right?

Reply:
There's Enum.HasFlag but people usually prefer to mask directly since calling a method on and with an enum has performance issues.